### PR TITLE
Updating the package list with Python Cosmos GA

### PR DESCRIPTION
**Pull request by kaerm**
waiting for whenever the package is out 

cc: @annatisch

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -2,7 +2,7 @@
 "App Configuration","azure-appconfiguration","1.0.0","","appconfiguration"
 "Cognitive Search","azure-search-documents","","1.0.0b3","search"
 "Core","azure-core","1.5.0","","core"
-"Cosmos","azure-cosmos","","4.0.0b6","cosmos"
+"Cosmos","azure-cosmos","4.0.0","","cosmos"
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:5]: This version hasn't been released yet https://pypi.org/project/azure-cosmos/#history so we shouldn't update it until it does. Once it gets released I expect my daily automation will update this.

Comment (kaerm) [reply _data/releases/latest/python-packages.csv:5]: Moving from preview to GA works?

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:5]: Yes it should.

Comment (kaerm) [reply _data/releases/latest/python-packages.csv:5]: That's awesome! 
cc: @jongio for visibility

and I am closing this PR as it should be automatically updated

**Pull request closed without merging**